code/modules/vehicles/speedbike.dm

```diff
@@ -36,6 +36,363 @@
 	icon_state = "speedbike_red"
 	overlay_state = "cover_red"
 
+
+/obj/vehicle/space/speedbike/atmos
+	name = "prototype atmos vehicle"
+	desc = "This vehicle possesses unparalled utility for atmospherics containment and control"
+	icon_state = "atmo_bike"
+	overlay_state = "cover_atmo"
+	var/obj/machinery/portable_atmospherics/scrubber/huge/HS = null
+	var/obj/item/weapon/extinguisher/vehicle/EX = null
+	var/obj/machinery/portable_atmospherics/canister/CAN = null
+	var/loaded = TRUE
+
+/obj/vehicle/space/speedbike/atmos/New()
+	. = ..()
+	HS = new /obj/machinery/portable_atmospherics/scrubber/huge(src)
```

Comment:
single/two letter variable names are only appropriate within local scopes, not on objects.